pkg/apis/extensions/v1beta1/types.go

```diff
@@ -967,10 +967,11 @@ type PodSecurityPolicySpec struct {
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty" protobuf:"varint,14,opt,name=readOnlyRootFilesystem"`
 	// SeccompProfiles lists the allowed profiles that may be set for the pod or
-	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
-	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
-	// used to generate a value for a pod the first non-wildcard profile will be used as
-	// the default.
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles
+	// are allowed to be set and that a default (docker/default) will be set by the strategies.
+	// A non-empty value contains the list of allowed profiles and the first non-wildcard (*) should
```

Comment:
Sorry if wasn't clear. Second try ;-)

A string like "docker/default" is not a good idea because it depends on the container runtime. I was thinking above that it might be better to use "no annotation" as marker for the default. If the container runtime does not find any annotation, it will just apply the default (which had been "unconfined" before and should be the docker default profile, which had been the consensus in the discussion).